helpers/auth/authentication.py

```diff
@@ -91,9 +93,10 @@ def save_user(self):
             pass
         return True
 
-    def user_roles(self, *expected_args):
+    def user_roles(self, *expected_args):  # noqa: C901
         """ User roles """
 
+        @pytest.mark.skip(reason="no way of currently testing this")
```

Comment:
Nice work @michael-basweti.

From pytest documentation, this decorator is used to mark tests and not actual implementations.
Please share your reason for placing it here.

Reply:
I was just testing something here, will remove it. The users.json will be shared on slack